datadog_checks_base/tests/base/utils/test_http.py

```diff
@@ -2042,18 +2042,21 @@ def test_session_timeout(self):
 
 class TestAIAChasing:
     def test_incomplete_chain(self):
+        # Protocol 1.2 is allowed by default
         http = RequestsWrapper({}, {})
         http.get("https://incomplete-chain.badssl.com/")
 
-    def test_tls_protocol(self, caplog):
-        http = RequestsWrapper({'tls_protocols_allowed': ['TLSv1.2']}, {})
-        http.get("https://incomplete-chain.badssl.com/")
-
+    def test_cant_allow_unknown_protocol(self, caplog):
         with caplog.at_level(logging.WARNING):
-            http = RequestsWrapper({'tls_protocols_allowed': ['unknown']}, {})
+            RequestsWrapper({'tls_protocols_allowed': ['unknown']}, {})
```

Comment:
Should this still have 
```suggestion
            http = RequestsWrapper({'tls_protocols_allowed': ['unknown']}, {})
```

Reply:
no, it was unused